modules/ingester/flush.go

```diff
@@ -275,6 +281,7 @@ func (i *Ingester) handleFlush(userID string, blockID uuid.UUID) (retry bool, er
 		start := time.Now()
 		err = i.store.WriteBlock(ctx, block)
 		metricFlushDuration.Observe(time.Since(start).Seconds())
+		metricFlushSize.Observe(float64(block.BlockMeta().Size))
```

Comment:
this is fine, but keep in mind that this is the trace data only. it does not take into account the bloom filters, meta or index.

these other objects are small in comparison, but non-zero.